### ISSUE TITLE
First node for CCF cluster failed to come up in CACI

### ISSUE THREAD

**Issue by gaurav137** [microsoft/CCF#6465]
Hit this issue on CACI where the start node itself has not come up. Looking at the container logs I see:
`[fail ] node/quote_endorsements_client.h:136 | Giving up retrying fetching attestation endorsements from 10.92.0.12 after 3 attempts`

Filing the bug to track this issue.

**Comment by gaurav137** [comment]
Perhaps logic can retry more and/or give a longer gap between the retries. The above logs show very fast retries.

**Comment by achamayou** [comment]
@gaurav137 I think what we want here mainly is:

1. More retries.
2. Node shut down if all attempts fail.

Does that make sense to you?

**Comment by gaurav137** [comment]
@achamayou yes sounds good. I'd also suggest to have the retry attempts/timeouts configurable via the cchost config file and not baked into the code.

**Comment by achamayou** [comment]
@gaurav137 yes, please see #6478
